funflow-examples/makefile-tool/src/Main.hs

```diff
@@ -11,6 +11,7 @@ module Main where
 
 import Control.Arrow
 import Control.Exception (SomeException (..))
+import Control.Kernmantle.Caching (caching)
```

Comment:
While this works, we should probably use the re-exported `caching` for consistency with the tutorials.

```suggestion
import Funflow (caching)
```